FordLabs/PeopleMover
# 161 preview message

## Issue
Resolves 161

## What was done
- [x] Researched OpenGraph
- [x] Set up metatags in index file to show preview information
- [x] Tested in feature branch  

## How to test
1. Wait until this is deployed to prod
2. Copy link to any environment
3. Paste into Slack, Yammer, Webex Teams
4. Note that a link preview appears in each location

## Accessiblity Criteria

### Manual Checks
